mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/UtbetalingService.kt

```diff
@@ -81,6 +93,75 @@ class UtbetalingService(
             }
     }
 
+    private suspend fun getPersoner(deltakerIdenter: List<NorskIdent>): Map<PdlIdent, Pair<HentPersonBolkResponse.Person, GeografiskTilknytningResponse?>> {
+        val identer = deltakerIdenter
+            .map { ident -> PdlIdent(ident.value) }
+            .toNonEmptySetOrNull()
+            ?: return mapOf()
+
+        return pdl.hentPersonOgGeografiskTilknytningBolk(identer).getOrElse {
+            throw StatusException(
+                status = HttpStatusCode.InternalServerError,
+                detail = "Klarte ikke hente informasjon om deltakere for kostnadsfordeling",
```

Comment:
Kanskje denne kan være mer generisk siden metoden potensielt i fremtiden kan bli brukt for annet enn bare kostnadsfordeling. Feks. Klarte ikke hente informasjon om personer og geografisk tilknytning?